### ISSUE TITLE
`limit` field should not raise a linter error

### ISSUE THREAD

**Issue by jmwski** [googleapis/api-linter#965]
#### Environment details

  - Programming language: proto3
  - OS: N/A
  - Language runtime version: N/A
  - Package version: N/A

I am internal g3 user [CL](cl/444322153).

#### Steps to reproduce

A linter warning is raised for the following field (the directive is required to silence this warning):

```
    // (-- api-linter: core::0140::prepositions=disabled
    // aip.dev/not-precedent: Consistent with an already public api. --)
    optional int32 limit = 3;
```

a) limit is both a noun and a verb, so makes sense on its own.
b) we aren't using it as a preposition of another word, we really are describing a Limit. 
c) Limit isn't on the official list of forbidden words: [https://linter.aip.dev/140/prepositions](https://www.google.com/url?q=https://linter.aip.dev/140/prepositions&sa=D&source=docs&ust=1651157830083835&usg=AOvVaw3q2UrutZrPG0mlvmXfWdTs)
